### ISSUE TITLE
[New Feature]: Adding Github/socials links and contact us form at the end of  About page

### ISSUE THREAD

**Issue by ananya-dhiman** [GameSphere-MultiPlayer/Physi-c-Tech#104]
### 🎮 feature Request

The addition of GitHub link and socials can help users reach out, the contact form can be added to act as feedback or to connect through email.

### Point down the features

A GitHub Link
Socials Link (If necessary)
A contact us form (Connect through email)

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE]() of this project.

**Comment by thevirengarg** [comment]
@ananya-dhiman You can work on it. For email you need to contact the PA @Durgesh4993. And add socials link of GameSphere. Discord link for the community is- https://discord.gg/BjzSZq8GnN

**Comment by ananya-dhiman** [comment]
@thevirengarg I tried to push my changes on a new branch contact but it shows this
![image](https://github.com/GameSphere-MultiPlayer/Physi-c-Tech/assets/145692647/14d9cb5f-a256-4d70-92e6-b087d2f9c831)
Please help.
I also didn't receive an email from @Durgesh4993 yet so please forward the message. I was planning to link the form through formsubmit.co so it will require verification from the receiver's end so please suggest how to go about it.

**Comment by thevirengarg** [comment]
@ananya-dhiman Did you fork and clone the repository?
Contact @Durgesh4993 via discord.
https://discord.gg/BjzSZq8GnN

**Comment by ananya-dhiman** [comment]
Yes ofcourse I followed all the steps and made upstream then made a new branch contact but can't push changes. So this isn't an access denied problem?

**Comment by ananyag309** [comment]
Hello, I'm interested in working on this issue. Please assign it to me.

**Comment by kunwarnaunihalsingh** [comment]
hey, please assign this project to me since I believe that my skills in HTML, CSS and JS make me an ideal fit to contribute for your project.

**Comment by thevirengarg** [comment]
@ananyag309 @kunwarnaunihalsingh This issue is currently assigned to @ananya-dhiman. You can create your own issues and i will assign you those issues.
